modules/ipfs-cpinner-client/README.md

```diff
@@ -55,17 +55,17 @@ const credentials = await client.get({ did, key })
 ```typescript
 import DataVaultWebClient from '@rsksmart/ipfs-cpinner-client'
 
-const client = new DataVaultWebClient({ serviceUrl, did, signer, serviceDid })
+const client = new DataVaultWebClient({ serviceUrl, did, rpcPersonalSign, serviceDid })
```

Comment:
We could just remove the setup part from each operation

Reply:
I included it because I wanted to show that the optional params are required for certain operations